src/WebJobs.Extensions.DocumentDB/Trigger/CosmosDBTriggerAttributeBindingProvider.cs

```diff
@@ -110,6 +114,16 @@ public CosmosDBTriggerAttributeBindingProvider(INameResolver nameResolver, Docum
             return new CosmosDBTriggerBinding(parameter, documentCollectionLocation, leaseCollectionLocation, leaseHostOptions);
         }
 
+        internal static TimeSpan ResolveTimeSpanFromMilliseconds(TimeSpan baseTimeSpan, int? attributeValue)
```

Comment:
Rather than support `int`, could you make the properties `TimeSpan` so you didn't have to have this method?

Reply:
I tried, but if you try to create a function with this signature:
`CosmosDBTrigger("mydb", "mycollection", FeedPoolDelay=TimeSpan.FromSeconds(5))`

It won't compile. There is a CLR restriction that only primitive types can be used.